_opensearch/rest-api/snapshots/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Snapshot APIs
+parent: REST API reference
+has_children: true
+nav_order: 6
+redirect_from:
+  - /opensearch/rest-api/document-apis/
+---
+
+# Snapshot APIs
+
+The snapshot APIs allow you to manage snapshots and snapshot repositories.
```

Comment:
Capitalize Snapshot APIs

Reply:
Should we add some more info here? At least a TOC with all operations, maybe grouped by whether they pertain to snapshot or repository.

Reply:
A TOC is automatically generated and order is based on the nav_order of each topic. I will revisit the order to group by snapshot and repository. Thank you for this suggestion!